libs/smart-contracts/src/utils/decimals.ts

```diff
@@ -0,0 +1,10 @@
+import BigNumber from 'bignumber.js';
+
+BigNumber.config({ EXPONENTIAL_AT: 20000 });
+
+export function addDecimal(value: BigNumber, decimals: number): BigNumber {
```

Comment:
question (non-blocking): seen these two also in `lib/react-helpers/src/lib/format/numbers.ts` in a slightly different variation, is there a natural habitat where these number related helpers should live in the monorepo?

Reply:
At the moment they live in react helpers, however this is a ts only library and that currently exports components as well so I didn't want this to be dependent on that. i think we need a ts only utils package for date/number/other ts only helpers, but didn't want the scope creep in this PR. I will ticket now.